{
  assertUnwrapped("try {\n" + "    int i;\n" + "} catch(RuntimeException e) {\n"+ "    int j;<caret>\n"+ "} catch(Exception e) {\n"+ "    int k;\n"+ "}\n","try {\n" + "    int i;\n" + "} <caret>catch(Exception e) {\n"+ "    int k;\n"+ "}\n");
}
